runtime/near-vm-runner/src/imports.rs

```diff
@@ -55,6 +55,7 @@ wrapped_imports! {
     // TODO #1903 rename to `block_height`
```

Comment:
It is unrelated to this PR, but given we break it here, should we also address this?

Reply:
We don't break here anything. This is additive change not requiring any changes on existing contracts. Switching to different function name would require to recompile all deployed contracts from my understanding @nearmax

Reply:
Yes, renaming the host function is harder and so we postpone it for the future. We probably will do a soft deprecation with warning and keep the old name forever.

Reply:
Do we want to add `block_height` field here or we don’t want to duplicate value for consistency reasons only and should just work around this on SDK level?

In any case, the comment and the issue will benefit if it gets updated with the current decision.

I’m sorry to hijack the PR discussion.

Reply:
Sure, let's add it here.

Reply:
@ilblackdragon ^

Reply:
I added details where they belong - https://github.com/nearprotocol/nearcore/issues/1903

If someone wants to actually add `block_height` - that indeed sounds like a separate PR.
Note also it should start with spec change.